gtsam/geometry/Pose3.cpp

```diff
@@ -354,6 +354,14 @@ Point3 Pose3::transformFrom(const Point3& point, OptionalJacobian<3, 6> Hself,
   return R_ * point + t_;
 }
 
+Matrix Pose3::transformFrom(const Matrix& points) const {
+  if (points.rows() != 3) {
+    throw std::invalid_argument("Pose3:transformFrom expects 3*N matrix.");
```

Comment:
Thanks for the PR, Frank. I personally would find it more intuitive to have (N,3) as input, rather than (3,N).

Reply:
I respect that, and realize many will feel that way as it's the numpy way. I don't understand why. Points are column vectors in all texts I know of, and replicating columns is much more natural, e.g.,  to multiply with matrices as the code above shows.  It is also used the convention elsewhere in GTSAM (unless it slipped in somewhere, in which case I'd like to deprecate it).